docroot/themes/humsci/humsci_basic/src/scss/utilities/_raised-cards.scss

```diff
@@ -5,5 +5,38 @@
       padding: hb-spacing-width('xs');
       box-shadow: hb-calculate-rems(1px) hb-calculate-rems(3px) hb-calculate-rems(17px) hb-get-global-color('gray-medium', $ht-traditional-globals);
     }
+
+    @include hb-themes(('airy', 'colorful')) {
+      box-shadow: hb-calculate-rems(1px) hb-calculate-rems(7.5px) hb-calculate-rems(35px) rgba(hb-get-global-color('black', $hc-colorful-globals), 0.12);
```

Comment:
maybe we can pop this in a general theme variable

Reply:
I'm not sure if that would be helpful. We probably need to take a look at the different box-shadows we have throughout the themes because we don't have any variables set but there are quite a few box-shadows being used. We could refactor into a mixin that would use the correct box-shadow for the theme but that level of refactor seems beyond the scope of this work.